lib/vjs-hls.js

```diff
@@ -30,7 +31,49 @@ var attachVideojsStreamrootProvider = function (window, videojs, Hls) {
         }
 
         function _onError(event, data) {
-            console.error('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details);
+            var error = {
+              message: ('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details),
+            };
+            console.error(error.message);
```

Comment:
this might cause a lot of log lines for non-fatal stuff, but just saw we are stripping this out in prod